releases/release-6.1.1.md

```diff
@@ -0,0 +1,183 @@
+    - Fix the issue that TiKV panics when performing type conversion for an empty string [#12673](https://github.com/tikv/tikv/issues/12673) @[wshwsh12 (Shenghui Wu)](https://github.com/wshwsh12)
+    - Fix the issue of inconsistent Region size configuration between TiKV and PD [#12518](https://github.com/tikv/tikv/issues/12518) @[5kbpers](https://github.com/5kbpers)
+    - Fix the issue that encryption keys are not cleaned up when Raft Engine is enabled [#12890](https://github.com/tikv/tikv/issues/12890) @[tabokie (Xinye Tao)](https://github.com/tabokie)
+    - Fix the panic issue that might occur when a peer is being split and destroyed at the same time [#12825](https://github.com/tikv/tikv/issues/12825) @[BusyJay](https://github.com/BusyJay)
+    - Fix the panic issue that might occur when the source peer catches up logs by snapshot in the Region merge process [#12663](https://github.com/tikv/tikv/issues/12663) @[BusyJay](https://github.com/BusyJay)
+    - Fix the issue of frequent PD client reconnection that occurs when the PD client meets an error [#12345](https://github.com/tikv/tikv/issues/12345) @[Connor1996](https://github.com/Connor1996)
+    - Fix potential panic when parallel recovery is enabled for Raft Engine [#13123](https://github.com/tikv/tikv/issues/13123) @[tabokie (Xinye Tao)](https://github.com/tabokie)
+    - Fix the issue that the Commit Log Duration of a new Region is too high, which causes QPS to drop [#13077](https://github.com/tikv/tikv/issues/13077) @[Connor1996](https://github.com/Connor1996)
+    - Fix rare panics when Raft Engine is enabled [#12698](https://github.com/tikv/tikv/issues/12698) @[tabokie](https://github.com/tabokie)
+    - Avoid redundant log warnings when proc filesystem (procfs) cannot be found [#13116](https://github.com/tikv/tikv/issues/13116) @[tabokie](https://github.com/tabokie)
+    - Fix the wrong expression of `Unified Read Pool CPU` in dashboard [#13086](https://github.com/tikv/tikv/issues/13086) @[glorv](https://github.com/glorv)
+    - Fix the issue that when a Region is large, the default [`region-split-check-diff`](/tikv-configuration-file.md#region-split-check-diff) might be larger than the bucket size [#12598](https://github.com/tikv/tikv/issues/12598) @[tonyxuqqi](https://github.com/tonyxuqqi)
+    - Fix the issue that TiKV might panic when Apply Snapshot is aborted and Raft Engine is enabled [#12470](https://github.com/tikv/tikv/issues/12470) @[tabokie](https://github.com/tabokie)
+    - Fix the issue that the PD client might cause deadlocks [#13191](https://github.com/tikv/tikv/issues/13191) @[bufferflies](https://github.com/bufferflies) [#12933](https://github.com/tikv/tikv/issues/12933) @[BurtonQin (Boqin Qin](https://github.com/BurtonQin)
+
++ PD
+
+    - Fix the issue that the online progress is inaccurate when label configurations of cluster nodes are invalid[#5234](https://github.com/tikv/pd/issues/5234) @[rleungx](https://github.com/rleungx)
```

Comment:
The contributor in Chinese is [rleungx (Ryan Leung)](https://github.com/rleungx)
```suggestion
    - Fix the issue that the online progress is inaccurate when label configurations of cluster nodes are invalid [#5234](https://github.com/tikv/pd/issues/5234) @[rleungx](https://github.com/rleungx)
```